sys/kern/vm_map.c

```diff
@@ -85,32 +70,15 @@ inline vm_map_entry_t *vm_map_entry_next(vm_map_entry_t *ent) {
   return TAILQ_NEXT(ent, link);
 }
 
-bool vm_map_address_p(vm_map_t *map, vaddr_t addr) {
-  return map && vm_map_start(map) <= addr && addr < vm_map_end(map);
-}
-
-bool vm_map_contains_p(vm_map_t *map, vaddr_t start, vaddr_t end) {
-  return map && vm_map_start(map) <= start && end <= vm_map_end(map);
-}
-
-vm_map_t *vm_map_lookup(vaddr_t addr) {
-  if (vm_map_address_p(vm_map_user(), addr))
-    return vm_map_user();
-  if (vm_map_address_p(vm_map_kernel(), addr))
-    return vm_map_kernel();
-  return NULL;
+static bool vm_map_contains_p(vaddr_t start, vaddr_t end) {
+  return USER_SPACE_BEGIN <= start && end <= USER_SPACE_END;
 }
```

Comment:
I'm not sure if name of this function should be prefixed by `vm_map_` since it check if given range belongs to userspace. Maybe something like `userspace_p(...)?